Source/Core/Core/PowerPC/JitArm64/JitArm64_BackPatch.cpp

```diff
@@ -405,10 +402,6 @@ bool JitArm64::HandleFault(uintptr_t access_address, SContext* ctx)
 	ctx->CTX_PC = (u64)slow_handler_iter->first;
 
 	// Wipe the top bits of the addr_register
-	if (flags & BackPatchInfo::FLAG_STORE &&
-	    !(flags & BackPatchInfo::FLAG_SIZE_F64))
-		ctx->CTX_REG(1) &= 0xFFFFFFFFUll;
-	else
-		ctx->CTX_REG(0) &= 0xFFFFFFFFUll;
+	ctx->CTX_REG(slow_handler_iter->second.addr_reg) &= 0xFFFFFFFFUll;
```

Comment:
Both addr and RS aren't modified within the fast-path, so which register do you want to fix here?